packages/studioCMS/src/integrations/studioCMSDashboard/utils/webVital.ts

```diff
@@ -11,14 +12,25 @@ export type WebVitalsResponseItem = {
 };
 
 export async function getWebVitals(): Promise<WebVitalsResponseItem[]> {
-	const webVitals: WebVitalsResponseItem[] = await db
-		.select()
-		.from(AstrojsWebVitals_Metric)
-		.catch(() => {
-			return [] as WebVitalsResponseItem[];
-		});
-	if (!webVitals) {
+	// Check if AstrojsWebVitals exists
+	if (!AstrojsWebVitals_Metric) {
+		logger.warn(
+			'@astrojs/web-vitals table not found.  If you have not installed the package you can disregard this warning.'
+		);
+		return [] as WebVitalsResponseItem[];
+	}
+
+	try {
+		// Execute the database query
+		const webVitals: WebVitalsResponseItem[] = await db.select().from(AstrojsWebVitals_Metric);
+
+		// Return the results if successful
+		return webVitals;
+	} catch (error) {
+		// Log the error for debugging purposes
+		logger.error(
+			`Error getting @astrojs/web-vitals table data.  If you have not installed the package you can disregard this error.\n - ${error}`
+		);
```

Comment:
Same issue here, though with the early return should you ever reach this code block if you don't have it installed?

Reply:
In theory it is possible for this error to get hit without the whole package installed, (mainly if you remove the @astrojs/web-vitals but dont remove the table with the data)